java/ql/src/Security/CWE/CWE-273/UnsafeCertTrust.qhelp

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>Java offers two mechanisms for SSL authentication - trust manager and hostname verifier (the later is checked by the <code>java/insecure-hostname-verifier</code> query). Trust manager validates the peer's certificate chain while hostname verification establishes that the hostname in the URL matches the hostname in the server's identification.</p>
```

Comment:
Perhaps add an article here? (Feel free to ignore if you don't agree.)
```suggestion
<p>Java offers two mechanisms for SSL authentication - trust manager and hostname verifier (the later is checked by the <code>java/insecure-hostname-verifier</code> query). The trust manager validates the peer's certificate chain while hostname verification establishes that the hostname in the URL matches the hostname in the server's identification.</p>
```

Reply:
I think the TrustManager part has also been removed from the query.

Reply:
True, but given that the next sentence clarifies the pattern this query is modeling, I wouldn't remove it (unless someone has a strong opinion in favor of doing it) since I don't think the additional context hurts here.